README.md

```diff
@@ -90,7 +94,46 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+Submariner installation can be done manually using a dedicated tool, `subctl`, or Helm charts, or
+by using its operator. All three methods are described here.
+
+## Installation using subctl
+
+`subctl` is a Submariner-specific tool developed to simplify Submariner installation. It drives
+[the operator](#installation-using-operator). You can see it in action in
+[this demo](https://youtu.be/cInmBXuZsU8).
+
+### Download
+
+`subctl` is maintained in the operator project; binaries are available for download from
+[the operator releases](https://github.com/submariner-io/submariner-operator/releases).
+Download the latest binary for your operating system and architecture, and install it on your path;
+it has no external dependencies.
```

Comment:
```suggestion
It has no external dependencies.
```

Reply:
This is part of the sentence on the previous line.